lib/src/isolate.dart

```diff
@@ -69,6 +70,11 @@ class VMIsolateRef {
   Stream<VMIsolateRef> get onUpdate => _onUpdate;
   Stream<VMIsolateRef> _onUpdate;
 
+  /// A broadcast stream that emits a new reference to this isolate every time
+  /// it has been reloaded.
```

Comment:
Explain what "reload" means in this context.

Why is this returning a new isolate ref? Does a reload change the metadata at all?

Also add a note indicating what protocol version this was added in.